dbt_sugar/core/task/audit.py

```diff
@@ -0,0 +1,243 @@
+"""Document Task module."""
+from typing import Any, Dict, List
+
+from rich.console import Console
+from rich.table import Table
+
+from dbt_sugar.core.clients.yaml_helpers import open_yaml
+from dbt_sugar.core.flags import FlagParser
+from dbt_sugar.core.logger import GLOBAL_LOGGER as logger
+from dbt_sugar.core.task.base import COLUMN_NOT_DOCUMENTED, BaseTask
+
+console = Console()
+NUMBER_COLUMNS_TO_PRINT_PER_ITERACTION = 5
+
+
+class AuditTask(BaseTask):
+    """Audit Task object.
+
+    Holds methods and attrs necessary to audit a model or a DBT project.
+    """
+
+    def __init__(self, flags: FlagParser) -> None:
```

Comment:
I think we need to read the config and possibly the dbt project here so that we can access the syrup and know which dbt projects are under scope